.changeset/eighty-rockets-yell.md

```diff
@@ -27,4 +27,4 @@ To better differentiate between imports from `@comet/admin` and `@mui/material`,
 - `MenuItemRouterLink` → `MainNavigationItemRouterLink`
 - `MenuItemRouterLinkProps` → `MainNavigationItemRouterLinkProps`
 
-Add `useMainNavigation()` hook to use instead of `MainNavigationContext`
+Remove export of `MainNavigationContent`. Use `useMainNavigation()` hook instead.
```

Comment:
I think when mentioning the removal, it should use the previous name, as the new one didn't exist yet. 
Also you can remove the mention of the rename for the removed item, as this won't be relevant. 

```
Remove `MenuContext`, use the `useMainNavigation()` hook instead.
```